core/commands/storage/announce.go

```diff
@@ -70,13 +88,48 @@ $ btfs storage announce --host-storage-price=1000000`,
 			}
```

Comment:
The original check is no longer valid, should remove this check because the other parameters can still be set even storage host is not enabled.